README.md

```diff
@@ -103,7 +103,7 @@ This is a list of the copied files:
 * `internal/fakecgo/abi_*.h` from package `runtime/cgo`
 * `internal/fakecgo/asm_GOARCH.s` from package `runtime/cgo`
 * `internal/fakecgo/callbacks.go` from package `runtime/cgo`
-* `internal/fakecgo/go_GOOS_GOARCH.go` from package `runtime/cgo`
+* `internal/fakecgo/go_GOOS.go` from package `runtime/cgo`
```

Comment:
I'd like to separate this item from the list and put a sentence indicating that `go_GOOS.go` files are modified from `gcc_GOOS_GOARCH.go` files below the list.